Workbooks/Traffic Analytics/Unified/ApplicationPorts.workbook

```diff
@@ -0,0 +1,955 @@
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 9,
+            "content": {
+              "version": "KqlParameterItem/1.0",
+              "parameters": [
+                {
+                  "id": "3a316914-57a8-40de-b308-1d5abc01ec76",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "QueryType",
+                  "type": 1,
+                  "isHiddenWhenLocked": true,
+                  "criteriaData": [
+                    {
+                      "criteriaContext": {
+                        "leftOperand": "FlowType",
+                        "operator": "==",
+                        "rightValType": "static",
+                        "rightVal": "NSG",
+                        "resultValType": "static",
+                        "resultVal": "{isNSGTopologySchemaPresent},NSG"
+                      }
+                    },
+                    {
+                      "criteriaContext": {
+                        "leftOperand": "FlowType",
+                        "operator": "==",
+                        "rightValType": "static",
+                        "rightVal": "VNet",
+                        "resultValType": "static",
+                        "resultVal": "{isVNetTopologySchemaPresent},VNet"
+                      }
+                    },
+                    {
+                      "criteriaContext": {
+                        "operator": "Default",
+                        "resultValType": "static",
+                        "resultVal": "{isNSGTopologySchemaPresent},NSG,{isVNetTopologySchemaPresent},VNet"
+                      }
+                    }
+                  ],
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                },
+                {
+                  "id": "b1715201-f711-4a92-bf97-4720b2e295c7",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "SubscriptionQueryToRun",
+                  "type": 1,
+                  "isHiddenWhenLocked": true,
+                  "criteriaData": [
+                    {
+                      "criteriaContext": {
+                        "leftOperand": "QueryType",
+                        "operator": "==",
+                        "rightValType": "static",
+                        "rightVal": "true,NSG",
+                        "resultValType": "static",
+                        "resultVal": "AzureNetworkAnalytics_CL | extend timecheck = iff({timeInterval:seconds} < 86400, TimeGenerated between (datetime_add('day', -1, todatetime('{timeInterval:endISO}')) .. todatetime('{timeInterval:endISO}')), TimeGenerated between (todatetime('{timeInterval:startISO}') .. todatetime('{timeInterval:endISO}'))) | where SubType_s == 'StatusMessage' and ComponentType_s == 'Topology' | distinct Subscription_g, SubscriptionName_s | project Subscription = Subscription_g, SubscriptionName = SubscriptionName_s, selected = 1"
```

Comment:
WHat is startISO and endISO? Is it indicating time selected in the dropdown?
What will be the time zone? Will it be in IST if I query here in India or is it always UTC?